test/jackdaw/test/commands/write_test.clj

```diff
@@ -0,0 +1,94 @@
+(ns jackdaw.test.commands.write-test
+  (:require
+   [jackdaw.test.commands.write :as write]
+   [jackdaw.test.transports :as trns]
+   [jackdaw.test.transports.kafka]
+   [jackdaw.test.serde :as serde]
+   [jackdaw.test :refer [test-machine]]
+   [clojure.test :refer :all]
+   [jackdaw.serdes.avro.schema-registry :as reg]))
+
+(def schema-registry-config
+  {:avro.schema-registry/client (reg/mock-client)
+   :avro.schema-registry/url    "localhost:8081"})
+
+(def resolver (serde/local-serdes-resolver schema-registry-config))
```

Comment:
The above 5 lines are repeated in a few places so we should perhaps try to factor it out somewhere better.